user_manager/api/v1/views.py

```diff
@@ -78,6 +79,19 @@ def authentication_classes(self):  # pragma: no cover
         ]
 
 
+class BulkCreateMixin(object):
```

Comment:
Thanks for making this a mixin! Will be useful for the bulk manager API if we ever add that.

Reply:
Exactly, so we prevent redundant logic later.